assets/dist/scss/components/_components.articles copy.scss

```diff
@@ -0,0 +1,110 @@
+////////////////////////////////
+// !CORE / COMPONENTS / ARTICLES
+////////////////////////////////
+
+
+.dcf-article-body dl {
+  @include pl-6;
+}
+
+
+.dcf-article-body dl,
+.dcf-article-body ol,
+.Dcf-article-body ul {
+  margin-bottom:0;
+}
+
+
+.dcf-article p,
+.dcf-article dl,
+.dcf-article ol,
+.dcf-article ul,
+.dcf-article blockquote {
+
+  + p,
+  + dl,
+  + ol,
+  + ul {
+    @include mt-1;
+  }
+
+  + p {
+    text-indent: #{ms(4)}em; // 1.77em
+  }
+
+}
+
+
+.dcf-article-body p:not(.dcf-lead) {
+  margin-bottom: 0;
+}
+
+
+.dcf-author-img img {
+  display: block;
+  width: #{ms(8)}em; // 3.16em
+}
+
+
+@include mq(sm, max, width) {
+
+  .dcf-article-share li {
+    display: inline-flex;
+    margin-bottom: 0;
+    @include mr-4;
+  }
+
+}
+
+
+@include mq(sm, min, width) {
+
+  .dcf-article-share div[role="heading"] {
```

Comment:
Just a comment, but I'd avoid and not encourage using role="heading". An element like `<h2>` is much better.

Reply:
@skoolbus39 want to take a stab at this?

Reply:
Yep, this is from early development, before we made the change from roles to heading elements in the footer. The article component isn't currently a high priority, but thanks for putting this particular issue on my radar. I'll make sure it gets changed.